Danki2/Assets/Scripts/Actor/Mortal/Mortal.cs

```diff
@@ -1,7 +1,28 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public abstract class Mortal : Actor
+﻿public abstract class Mortal : Actor
 {
+    public int Health { get; private set; }
+
+    public void ModifyHealth(int healthChange)
+    {
+        Health += healthChange;
+    }
+
+    protected override void Start()
+    {
+        base.Start();
+
+        Health = GetStat(Stat.Health);
```

Comment:
I think it might be a bit clearer if Stats.Health is renamed to Stats.MaxHealth.